packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -167,6 +167,8 @@ import {
 import { extractValue } from './ExtractValue';
 import { InstanceSettings } from './InstanceSettings';
 import type { ExtendedValidationResult, IResponseError } from './Interfaces';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Could we move the functions `PollContext` needs from this file to a separate file so we wouldn't have to introduce a cycle?

Reply:
I haven't moved many functions out of `NodeExecutionFunctions` yet. Once I have extractd all the context objects, those functions will be moved to `BaseContext` as private methods. We can then remove this eslint exception.